xtrabackup/base/Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN apt-get update \
 
 RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 9334A25F8507EFA5
 
-RUN echo 'deb https://repo.percona.com/apt bullseye main' > /etc/apt/sources.list.d/percona.list
+RUN echo 'deb https://repo.percona.com/apt stretch main' > /etc/apt/sources.list.d/percona.list
```

Comment:
should use the version argument so that we can build with different debian versions